docs/index.rst

```diff
@@ -55,6 +55,7 @@ Table Of Contents
    tutorial/index
    architecture
    install
+   images
```

Comment:
this is definitely not a level-one topic, can be placed under functions

Reply:
@yaronha:  Any preference where under Functions?

Reply:
Now under Functions > Kubernetes Jobs & Images